src/providers/spatialite/qgsspatialiteprovider.cpp

```diff
@@ -5820,52 +5820,54 @@ bool QgsSpatiaLiteProvider::getTableSummaryAbstractInterface( gaiaVectorLayerPtr
 
 bool QgsSpatiaLiteProvider::getTableSummary()
 {
-  int ret;
-  int i;
-  char **results = nullptr;
-  int rows;
-  int columns;
-  char *errMsg = nullptr;
 
-  QString sql = QStringLiteral( "SELECT Count(1)%1 FROM %2" )
-                .arg( mGeometryColumn.isEmpty() ? QString() : QStringLiteral( ",Min(MbrMinX(%1)),Min(MbrMinY(%1)),Max(MbrMaxX(%1)),Max(MbrMaxY(%1))" ).arg( QgsSqliteUtils::quotedIdentifier( mGeometryColumn ) ),
-                      mQuery );
+  if ( mGeometryColumn.isEmpty() )
+  {
+    mLayerExtent.setMinimal();
+    return true;
+  }
+
+  QString sql = QStringLiteral(
+    "SELECT Count(1), Min(MbrMinX(%1)), Min(MbrMinY(%1)), Max(MbrMaxX(%1)), Max(MbrMaxY(%1)) FROM %2"
+  ) .arg( QgsSqliteUtils::quotedIdentifier( mGeometryColumn ), mQuery );
 
   if ( !mSubsetString.isEmpty() )
   {
     sql += " WHERE ( " + mSubsetString + ')';
   }
 
-  ret = sqlite3_get_table( sqliteHandle( ), sql.toUtf8().constData(), &results, &rows, &columns, &errMsg );
+  char **results = nullptr;
+  int rows;
+  int columns;
+  char *errMsg = nullptr;
+
+  int ret = sqlite3_get_table( sqliteHandle( ), sql.toUtf8().constData(), &results, &rows, &columns, &errMsg );
```

Comment:
Should this be

```suggestion
  int ret = sqlite3_get_table( sqliteHandle( ), sql.toUtf8( ).constData( ), &results, &rows, &columns, &errMsg );
```

Reply:
is there a policy about these parens ? I see no space used below, maybe I shoudl remove it from the first element too